README.md

```diff
@@ -24,14 +26,4 @@ Repo contains init travis.yml and setup to run testing on push/PR. However, **no
 Change `name: "meta.class.matlab"` to `name: ""`. This prevents atom from improperly coloring of all class contents.
 
 1. Add in function highlighting from functions.cson
-MATLAB-Language-grammar does not highlight MATLAB functions like the TextMate grammar. The file `functions.cson` contains the functions from the TextMate grammar to be highlighted. Just add the components of this file to the corresponding places in `grammars/m.cson`.
-
-## Testing/Verifying Grammar Changes
```

Comment:
More confusion (on me) and apologies again! 

I liked this section as well as 1) moving test file to root directory and 2) including the image example in the README.

My Suggested TO-DO:
- [x] Add image of highlighting.m to README
- [x] Move highlighting.m to root directory. Rename `highlighting-test.m`
- [ ] Provide (keep) instructions in README about how to use highlighting-test.m. (Might be simpler than current if you want. But I like what you added here)

Reply:
Added comments back in 7d606934f3f294a978c7d3126e7f521b6c0a48ce. Also clarified the function highlighting step since we removed `functions.cson`.